src/prefect/cli/agent.py

```diff
@@ -208,6 +224,8 @@ def start(
                                         e.g. `-p /root/my_scripts -p /utilities`
         --show-flow-logs, -f        Display logging output from flows run by the agent
                                     (available for Local and Docker agents only)
+        --hostname-label            Add hostname to the Agent's labels
+        --storage-labels            Add all available storage label to the Agent
```

Comment:
Since both of these default to true we should probably document here that the `--no-XXXX` option will disable it

Reply:
Added!